diff --git a/node_modules/@angular-devkit/schematics-cli/bin/schematics.js b/node_modules/@angular-devkit/schematics-cli/bin/schematics.js
index 78727ae..68de5d0 100644
--- a/node_modules/@angular-devkit/schematics-cli/bin/schematics.js
+++ b/node_modules/@angular-devkit/schematics-cli/bin/schematics.js
@@ -33,7 +33,7 @@ const minimist = require("minimist");
  * @return {{collection: string, schematic: (string)}}
  */
 function parseSchematicName(str) {
-    let collection = '@schematics/schematics';
+    let collection = './';
     let schematic = str;
     if (schematic && schematic.indexOf(':') != -1) {
         [collection, schematic] = schematic.split(':', 2);
@@ -101,7 +101,7 @@ async function main({ args, stdout = process.stdout, stderr = process.stderr, })
     const { collection: collectionName, schematic: schematicName, } = parseSchematicName(argv._.shift() || null);
     const isLocalCollection = collectionName.startsWith('.') || collectionName.startsWith('/');
     /** Gather the arguments for later use. */
-    const debug = argv.debug === null ? isLocalCollection : argv.debug;
+    const debug = argv['debug'] ? true : false;
     const dryRun = argv['dry-run'] === null ? debug : argv['dry-run'];
     const force = argv['force'];
     const allowPrivate = argv['allow-private'];
@@ -153,12 +153,12 @@ async function main({ args, stdout = process.stdout, stderr = process.stderr, })
                 logger.error(`ERROR! ${eventPath} ${desc}.`);
                 break;
             case 'update':
-                loggingQueue.push(core_1.tags.oneLine `
+                loggingQueue.push(core_1.tags.oneLine`
         ${core_1.terminal.white('UPDATE')} ${eventPath} (${event.content.length} bytes)
       `);
                 break;
             case 'create':
-                loggingQueue.push(core_1.tags.oneLine `
+                loggingQueue.push(core_1.tags.oneLine`
         ${core_1.terminal.green('CREATE')} ${eventPath} (${event.content.length} bytes)
       `);
                 break;
@@ -257,7 +257,7 @@ exports.main = main;
 * Get usage of the CLI tool.
 */
 function getUsage() {
-    return core_1.tags.stripIndent `
+    return core_1.tags.stripIndent`
   schematics [CollectionName:]SchematicName [options, ...]
 
   By default, if the collection name is not specified, use the internal collection provided
